ynh-dev

```diff
@@ -99,7 +102,7 @@ elif [ "$1" = "create-env" ]; then
 elif [ "$1" = "run" ]; then
 
     DOMAIN=$2
-    VERSION='stable'
+    VERSION='stable64'
```

Comment:
I proposed this for the sake of consistency and predictability (in my head when the architecture is not specified, it is 32 bits but I probably live 10 years in the past :) )